Management_System/Management_System/Entities/UserTaskEntity.cs

```diff
@@ -2,10 +2,12 @@
 
 namespace ManagementSystem.Entities
 {
-    public class UserTask
+    public class UserTaskEntity
     {
         public Guid UserId { get; set; }
         public Guid TaskId { get; set; }
+        public UserEntity User { get; set; } = null!;
+        public TaskEntity Task { get; set; } = null!;
         public TaskRole Role { get; set; }
```

Comment:
What is "Role" doing in the conjoint table?

Reply:
To define user's role for task (Reviewer, Assignee)